pymeasure/display/inputs.py

```diff
@@ -80,18 +80,14 @@ def parameter(self):
         return self._parameter
 
 
-class StringInput(QtGui.QLineEdit, Input):
+class StringInput(Input, QtGui.QLineEdit):
```

Comment:
If you have the time to comment, I'd be curious why the reordered superclasses support multiple inheritance in pyside2 better?

Reply:
You can have a look at https://github.com/pymeasure/pymeasure/pull/383#discussion_r614554704

Essentially using `super ` and multiple inheritance requires that the classes are designed to work cooperatively.
This boils down to the fact that `__init__ `method should call `super`. In fact the class `Input `does this, but the widget classes from `Pyside2 `apparently not.

Reply:
Ah, that makes sense, some shenanigans with `super` were the primary culprit I suspected.